.gitlab-ci.yml

```diff
@@ -55,14 +55,18 @@ Testing:
     - /usr/libexec/tests/osbuild-composer/image_tests.sh
   parallel:
     matrix:
+      # - RUNNER:
+      #     - aws/fedora-33-x86_64
+      #     - aws/fedora-33-aarch64
+      #   INTERNAL_NETWORK: "true"
+      # - RUNNER:
+      #     - aws/rhel-8-x86_64
+      #   REGISTER: "true"
+      #   DISTRO_CODE: "rhel_8"
+      #   INTERNAL_NETWORK: "true"
       - RUNNER:
-          - aws/fedora-33-x86_64
-          - aws/fedora-33-aarch64
-        INTERNAL_NETWORK: "true"
-      - RUNNER:
-          - aws/rhel-8-x86_64
-        REGISTER: "true"
-        DISTRO_CODE: "rhel_8"
+          - aws/rhel-8.5-x86_64
+          - aws/rhel-9.0-beta-nightly-x86_64
```

Comment:
If you update to 64113e98855ec44835b7534716ff44e1370d2241 you can use the rhel-8.5-nightly-x86_64 and rhel-8.4-ga-x86_64 references.

Reply:
Thanks! I'll try that.